community/terra/qis_adv/quantum_walk.ipynb

```diff
@@ -0,0 +1,570 @@
+      "Requirement already satisfied: asn1crypto>=0.21.0 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (0.24.0)\n",
+      "Requirement already satisfied: pycparser in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cffi!=1.11.3,>=1.7->cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (2.19)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install qiskit==0.7.1\n",
+    "!pip install qiskit_aer==0.1.1\n",
+    "!pip install IBMQuantumExperience"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Implementation of Quantum Walks on graph  for IBM Q\n",
+    "This notebook is based on the paper of  B L Douglas and J B Wang, \"Efficient quantum circuit implementation of quantum walks\", arXiv:0706.0304 [quant-ph]."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Contributors\n",
+    "Jordan Kemp(University of Chicago), Shin Nishio(Keio University), Ryosuke Satoh(Keio University), Desiree Vogt-Lee(University of Queensland), and Tanisha Bassan(The  Knowledge Society)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction: challenges in implementing quantum walk\n",
+    "\n",
+    "There are so  many types of quantum walks. Walker can walk on n-dimensional space or any limited graphs. First we talk about the concept and dynamics of Quantum and Classical random walk. After that we show the implementation of Quantum Walk on cycle graph."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Random walk \n",
+    "Random walk is a dynamics which is randomly time evolving system. Figure shows a simple type of random walk. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"../images/quantum_walk/random_walk.png\" alt=\"Note: In order for images to show up in this jupyter notebook you need to select File => Trusted Notebook\" width=\"300 px\" align=\"center\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The dynamics can be regarded as a simple algorithm as shown.\n",
+    "1. There is a $n$-dimension(in this case, one for simple) space and a walker start at the point $x=0$\n",
+    "2. take a step either forward (toward $+x$) or backward(toward $-x$) \n",
+    "in 2., the choice is to be made randomly(ex: coin-flip). We call this \"Coin Operator\". \n",
+    "\n",
+    "In this system, $p+q = 1$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quantum walk \n",
+    "Quantum walk is \"quantum version\" of random walk. This means theoin function can be Unitary gate($U(2)$) which is non-random and reversible. \n",
```

Comment:
theoin?

Reply:
I'm sorry, this is just typo.
theoin -> the coin